src/Chirp.Core/Classes/CheepRepository.cs

```diff
@@ -99,11 +99,13 @@ public List<CheepDTO> GetCheeps(int page)
 
     public List<Author> getFollowedInCheeps(Author follower)
     {
-        var authors = _dbContext.Follows
+        List<Author> authors = _dbContext.Follows
             .Where(f => f.Follower == follower)
             .Select(f => f.Followed)
             .Distinct()
-            .ToList();
+            .ToList()!;
```

Comment:
Looks really good in general, this I am not sure about however? Exclam is a way to guarantee not null? But is that good code? Can't we just do errorhandling instead?

Reply:
The exclamation point essentially says "trust me bro, its not null"

in commit [17f7755](https://github.com/ITU-BDSA2024-GROUP9/Chirp/pull/54/commits/17f77557e99136d42f93bb7f5f21d2c7c5d0b3c8) we ensure that it won't return null authors. It could potentially still return an empty list though.